deploy/ocs-operator/manifests/provider-role.yaml

```diff
@@ -15,6 +15,7 @@ rules:
   - ceph.rook.io
   resources:
   - cephfilesystemsubvolumegroups
+  - cephblockpoolradosnamespaces
```

Comment:
remove blockpool RBAC like Create/Delete and add this at the top of the function before `Reconcile` like below and let these gets auto generated.

// +kubebuilder:rbac:groups=ceph.rook.io,resources=cephblockpools,verbs=get;list;watch;create;update;delete

Reply:
I don't think so, rbac for provider-sa should be hand written.

Reply:
Yes correct but the controller need to be adjusted for the above RBAC but it can be a separate PR.

Reply:
> but it can be a separate PR.
- no, will include in this only, it's hard to track many, I saw StorageClaim got ready after for block when I updated the rbac for provider-sa, for controller I believe there's a large net like `can do anything for all resources in ceph.rook.io`.

will recheck though.

Reply:
verified, rbac for controllers already exist here
https://github.com/red-hat-storage/ocs-operator/blob/b2c823679e4bd7c633483df193af80be11d0aced/config/rbac/role.yaml#L39-L51

Reply:
done.